src/main/java/com/project/carsharingapp/controller/PaymentController.java

```diff
@@ -0,0 +1,58 @@
+package com.project.carsharingapp.controller;
+
+import com.project.carsharingapp.dto.payment.CreatePaymentSessionRequestDto;
+import com.project.carsharingapp.model.Payment;
+import com.project.carsharingapp.service.PaymentService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.net.URI;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Payment controller", description = "Endpoint for managing payments")
```

Comment:
```suggestion
@Tag(name = "Payment management", description = "Endpoint for managing payments")
```
To make a uniform style for controllers.